app/js/sumo_db.js

```diff
@@ -343,6 +343,13 @@
       endpoint += username + '/request_password_reset/';
 
       return request(endpoint, 'GET');
+    },
+
+    get_top_helpers: function() {
+      var endpoint = API_V2_BASE + 'user/weekly-solutions/';
```

Comment:
Is this request returning a maximum of 5 questions? Rather than slicing on the client side, it would be nice to have the smallest possible HTTP response.